Gemfile.lock

```diff
@@ -312,6 +312,3 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   w3c_validators
-
-BUNDLED WITH
-   1.10.5
```

Comment:
I have wanted to ask this for so long. Do I have to commit the Gemfile.lock file or not everytime it is updated locally ? I didn't see anyone doing it often so I didn't do it till now. But, this file gets during work.